src/v3/src/components/LaunchAuthenticatorButton/LaunchAuthenticatorButton.tsx

```diff
@@ -39,13 +40,23 @@ const LaunchAuthenticatorButton: UISchemaElementComponent<{
   } = uischema;
 
   const focusRef = useAutoFocus<HTMLButtonElement>(focus);
+  const {
+    userIdentifier,
+    setUserIdentifier,
+    data,
+    widgetProps: { features },
+  } = useWidgetContext();
 
   const handleClick: ClickHandler = async () => {
+    if (features?.engFastpassMultipleAccounts && data.identifier) {
+      // set userIdentifier in widget context to the current Username input field data
+      setUserIdentifier(encodeURIComponent(data.identifier as string));
```

Comment:
Just double-checking - is our ES6 polyfill configured to include this? Assuming we're using `core-js` then it should be included since it's part of most of the usual entry points: https://github.com/zloirock/core-js#url-and-urlsearchparams